FSI/two_flap_perp_2D/OpenFOAM-deal.II/plotDisplacement.sh

```diff
@@ -0,0 +1,13 @@
+#! /bin/bash
+gnuplot -p << EOF
+set grid
+set title 'Displacement of the Flap Tip'
+set xlabel 'Time [s]'
+set ylabel 'X-Displacement [m]'
+# set style line 1 lt 2 lw 6
+# set style line 1 lt 2 lw 6
+# set linestyle  2 lt 2 lc 1 # red-dashed
+set linestyle  1 lt 2 lc 1 # red-dashed
+plot "precice-Solid-watchpoint-flap_tip.log" using 1:4 title 'Top displacemement' with lines
```

Comment:
This will probably need to become:
```suggestion
plot "precice-Solid1-watchpoint-flap_tip.log" using 1:4 title 'Top displacemement' with lines
```

Reply:
And also extend it for Solid2, with a different line style/color.